manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -27,11 +27,38 @@ pilot:
   # Set to `type: RuntimeDefault` to use the default profile if available.
   seccompProfile: {}
 
+  # Additional container arguments
+  extraContainerArgs: []
+  # - --tlsCertFile=/etc/cert-manager/tls/tls.crt
+  # - --tlsKeyFile=/etc/cert-manager/tls/tls.key
+  # - --caCertFile=/etc/cert-manager/ca/root-cert.pem
+
   env: {}
 
   cpu:
     targetAverageUtilization: 80
 
+  # Additional volumeMounts to the istiod container
+  volumeMounts: []
+    # - mountPath: /etc/cert-manager/tls
```

Comment:
I'd remove these comments. Unless istio wants to promote the use of cert-manager it should be more agnostic. It's fine to point to istio documentation for integration with cert-manager here though

Reply:
I have removed the example comments.

Reply:
If/when this is merged, I'll update the issue in the cert-manager repo with the examples as a reference. Thanks